web/client/plugins/GeoStory.jsx

```diff
@@ -43,7 +43,7 @@ const GeoStory = ({
 );
 
 const storyThemeSelector = (state) => {
-    return settingsSelector(state)?.theme?.general || {};
+    return settingsSelector(state)?.theme || {};
```

Comment:
This change is fine but it has effect on the general theme picker background and text color.
eg; cascade component now gets a wrong style object
https://github.com/geosolutions-it/MapStore2/blob/c81244574a60bc5fcac312b95e75304847b6c9b3/web/client/components/geostory/layouts/Cascade.jsx#L120-L125

so in this case it needs to be changed in `storyTheme?.general` and the same should be done for all the storyTheme props passed to the nested component of the cascade component